backend/test_observer/main.py

```diff
@@ -18,14 +18,22 @@
 #        Omar Abou Selo <omar.selo@canonical.com>
 #        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
 
+import os
 
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+import sentry_sdk
+
 
 from test_observer.controllers.router import router
 
-app = FastAPI()
 
+SENTRY_DSN = os.getenv("SENTRY_DSN")
```

Comment:
two things:
- we should probably have two sentry environments one for staging and another for production (I can only see production on sentry dashboard)
- we need to pass this environment variable through terraform -> charm config

Reply:
I've noticed that other projects on sentry don't have anything for staging

Reply:
That's ok it doesn't hurt to have staging and we could make use of it

Reply:
I'm not sure if we care about errors on staging. And we also don't have production yet. Probably it makes sense to use it with staging for now and then for production only

Reply:
I can see uses for staging. Imagine we deploy something on staging and find an issue. Sentry will send us frontend errors happening on the client. This way we can debug an issue on staging before deploying to production

Reply:
Agree, it can sometimes be useful in staging also